modules/porous_flow/test/tests/heat_advection/tests

```diff
@@ -27,19 +26,27 @@
     threading = '!pthreads'
     issues = '#10426'
     design = 'PorousFlowFullySaturated.md porous_flow/numerical_diffusion.md porous_flow/kt_worked.md porous_flow/tests/heat_advection/heat_advection_tests.md'
-    requirement = 'PorousFlow shall correctly advect heat energy with a moving fluid, using no numerical stabilization, and the user should be able to activiate this stabilization using the PorousFlow Action system'
-  [../]
-
-  [./heat_advection_1d]
+    requirement = 'The system shall correctly advect heat energy with a moving fluid, using no numerical stabilization, and the user should be able to activiate this stabilization using the PorousFlow Action system.'
+  []
+  [heat_advection_1d]
     type = CSVDiff
     input = heat_advection_1d.i
     csvdiff = 'heat_advection_1d_csv_T_0010.csv heat_advection_1d_csv_T_0060.csv'
     threading = '!pthreads'
     issues = '#7984'
     design = 'PorousFlowHeatAdvection.md PorousFlowEnergyTimeDerivative.md porous_flow/tests/heat_advection/heat_advection_tests.md'
-    requirement = 'PorousFlow shall correctly advect heat energy with a moving fluid, using full-upwinding for numerical stabilization'
-  [../]
-  [./heat_advection_1d_fullsat]
+    requirement = 'The system shall correctly advect heat energy with a moving fluid, using full-upwinding for numerical stabilization.'
+  []
+  [heat_advection_1d_fv]
+    type = CSVDiff
+    input = heat_advection_1d_fv.i
+    csvdiff = 'heat_advection_1d_fv_csv_T_0061.csv'
+    threading = '!pthreads'
+    issues = '#7984 #21275'
+    design = 'PorousFlowHeatAdvection.md PorousFlowEnergyTimeDerivative.md porous_flow/tests/heat_advection/heat_advection_tests.md'
+    requirement = 'The system shall correctly advect heat energy with a moving fluid, using full-upwinding for numerical stabilization.'
```

Comment:
@cticenhour do we not flag duplicate requirements for porous flow?

Reply:
I made the requirements unique now